README.md

```diff
@@ -408,11 +415,19 @@ You will need the following things properly installed on your computer.
 
 ### Running Tests
 
-* `make test`
+```bash
+pipenv shell
+pipenv install --dev
```

Comment:
This really should be `pipenv sync --dev`.
`install` will re-resolve all the dependencies if packages have been updated, potentially leading to a problem. 
`sync` installs whatever is in `Pipfile.lock`.
https://pipenv.pypa.io/en/latest/advanced/#using-pipenv-for-deployments